Integrations/integration-SplunkPy.yml

```diff
@@ -168,11 +168,12 @@ script:
         incident["occurred"] = event["_time"]
         incident["rawJSON"] = json.dumps(event)
         labels = []
-        isParseNotableEventsRaw = demisto.params()['parseNotableEventsRaw']
-        if isParseNotableEventsRaw:
-            rawDict = rawToDict(event['_raw'])
-            for rawKey in rawDict:
-                labels.append({'type':rawKey,'value':rawDict[rawKey]})
+        if demisto.get(demisto.params(),'parseNotableEventsRaw'):
```

Comment:
I think you can just add a `default` for `get` function

Reply:
is it supported?
Anyway it doesn't matter since default is false